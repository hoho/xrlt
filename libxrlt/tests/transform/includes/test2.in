id:0, type:0, last:0, error:0, data:
id:0, type:0, last:0, error:0, data:
id:0, type:0, last:0, error:0, data:
id:0, type:0, last:0, error:0, data:
id:1, type:400, last:0, error:0, data:200
id:1, type:600, last:1, error:0, data:/some/href
id:0, type:0, last:0, error:0, data:
id:2, type:400, last:0, error:0, data:200
id:2, type:600, last:1, error:0, data:ST
id:0, type:0, last:0, error:0, data:
id:3, type:400, last:0, error:0, data:200
id:3, type:600, last:0, error:0, data:js
id:3, type:600, last:1, error:0, data:on
id:0, type:0, last:0, error:0, data:
id:4, type:400, last:0, error:0, data:200
id:4, type:600, last:1, error:0, data:true
id:0, type:0, last:0, error:0, data:
id:5, type:400, last:0, error:0, data:200
id:5, type:600, last:1, error:0, data:true
id:0, type:0, last:0, error:0, data:
id:6, type:400, last:0, error:0, data:200
id:6, type:600, last:1, error:0, data:
id:0, type:0, last:0, error:0, data:
id:7, type:400, last:0, error:0, data:200
id:7, type:600, last:1, error:0, data:hehe
id:8, type:400, last:0, error:0, data:200
id:8, type:600, last:1, error:0, data:vava
id:9, type:400, last:0, error:0, data:200
id:9, type:600, last:1, error:0, data:hoho
id:10, type:400, last:0, error:0, data:200
id:10, type:600, last:1, error:0, data:veve
id:11, type:400, last:0, error:0, data:200
id:11, type:600, last:1, error:0, data:pa;va
id:12, type:400, last:0, error:0, data:200
id:12, type:600, last:1, error:0, data:super body
id:0, type:0, last:0, error:0, data:
id:13, type:400, last:0, error:0, data:200
id:13, type:600, last:1, error:0, data:{"aa": "bb"}
id:0, type:0, last:0, error:0, data:
