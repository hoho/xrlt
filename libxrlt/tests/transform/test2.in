id:0, type:0, last:0, error: 0, data:
id:0, type:0, last:0, error: 0, data:
id:0, type:0, last:0, error: 0, data:
id:0, type:0, last:0, error: 0, data:
id:1, type:2, last:1, error: 0, data:/some/href
id:0, type:0, last:0, error: 0, data:
id:2, type:2, last:1, error: 0, data:ST
id:0, type:0, last:0, error: 0, data:
id:3, type:2, last:0, error: 0, data:js
id:3, type:2, last:1, error: 0, data:on
id:0, type:0, last:0, error: 0, data:
id:4, type:2, last:1, error: 0, data:true
id:0, type:0, last:0, error: 0, data:
id:5, type:2, last:1, error: 0, data:true
id:0, type:0, last:0, error: 0, data:
id:6, type:2, last:1, error: 0, data:
id:0, type:0, last:0, error: 0, data:
id:7, type:2, last:1, error: 0, data:hehe
id:8, type:2, last:1, error: 0, data:vava
id:9, type:2, last:1, error: 0, data:hoho
id:10, type:2, last:1, error: 0, data:veve
id:11, type:2, last:1, error: 0, data:pa;va
id:12, type:2, last:1, error: 0, data:super body
id:0, type:0, last:0, error: 0, data:
id:13, type:2, last:1, error: 0, data:{"aa": "bb"}
id:0, type:0, last:0, error: 0, data:
