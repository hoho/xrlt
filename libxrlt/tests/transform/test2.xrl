<?xml version="1.0"?>
<xrl:requestsheet xmlns:xrl="http://xrlt.net/Transform">
    <xrl:response>
        <xrl:include>
            <xrl:href>
                <xrl:include>
                    <xrl:href>/test/href/1</xrl:href>
                    <xrl:type>text</xrl:type>
                </xrl:include>
            </xrl:href>

            <xrl:type>
                <xrl:include>
                    <xrl:href>/test/href/2</xrl:href>
                    <xrl:type>text</xrl:type>
                </xrl:include>
            </xrl:type>

            <xrl:method>
                <xrl:include>
                    <xrl:href>/test/href/3</xrl:href>
                    <xrl:type>text</xrl:type>
                    <xrl:success select="concat('PO', /)" />
                </xrl:include>
            </xrl:method>

            <xrl:with-header>
                <xrl:name>
                    <xrl:include>
                        <xrl:href>/test/href/4</xrl:href>
                        <xrl:type>text</xrl:type>
                        <xrl:success select="/" />
                    </xrl:include>
                </xrl:name>
                <xrl:value>
                    <xrl:include>
                        <xrl:href>/test/href/5</xrl:href>
                        <xrl:type>text</xrl:type>
                        <xrl:success select="concat(text(), text())" />
                    </xrl:include>
                </xrl:value>
            </xrl:with-header>

            <xrl:with-header>
                <xrl:test>
                    <xrl:include>
                        <xrl:href>/test/href/6</xrl:href>
                        <xrl:type>text</xrl:type>
                    </xrl:include>
                </xrl:test>
                <xrl:name>
                    <xrl:include>
                        <xrl:href>/test/href/7</xrl:href>
                        <xrl:type>text</xrl:type>
                    </xrl:include>
                </xrl:name>
                <xrl:value>
                    <xrl:include>
                        <xrl:href>/test/href/8</xrl:href>
                        <xrl:type>text</xrl:type>
                    </xrl:include>
                </xrl:value>
            </xrl:with-header>

            <xrl:with-param>
                <xrl:test>
                    <xrl:include>
                        <xrl:href>/test/href/9</xrl:href>
                        <xrl:type>text</xrl:type>
                    </xrl:include>
                </xrl:test>
                <xrl:body>
                    <xrl:include>
                        <xrl:href>/test/href/10</xrl:href>
                        <xrl:type>text</xrl:type>
                    </xrl:include>
                </xrl:body>
                <xrl:name>
                    <xrl:include>
                        <xrl:href>/test/href/11</xrl:href>
                        <xrl:type>text</xrl:type>
                    </xrl:include>
                </xrl:name>
                <xrl:value>
                    <xrl:include>
                        <xrl:href>/test/href/12</xrl:href>
                        <xrl:type>text</xrl:type>
                    </xrl:include>
                </xrl:value>
            </xrl:with-param>

            <xrl:with-param name="papapa">
                <xrl:include>
                    <xrl:href>/test/href/13</xrl:href>
                    <xrl:type>text</xrl:type>
                </xrl:include>
            </xrl:with-param>

            <xrl:with-body>
                <xrl:test>
                    <xrl:include>
                        <xrl:href>/test/href/14</xrl:href>
                        <xrl:type>text</xrl:type>
                    </xrl:include>
                </xrl:test>
                <xrl:value>
                    <xrl:include>
                        <xrl:href>/test/href/15</xrl:href>
                        <xrl:type>text</xrl:type>
                    </xrl:include>
                </xrl:value>
            </xrl:with-body>

            <xrl:success select="/" />
        </xrl:include>
    </xrl:response>
</xrl:requestsheet>
