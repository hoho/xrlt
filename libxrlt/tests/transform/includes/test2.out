XRLT_STATUS_SUBREQUEST
sr id: 1
sr method: GET
sr type: TEXT
sr url: /test/href/1
sr query: (null)
sr body: (null)
XRLT_STATUS_SUBREQUEST
sr id: 2
sr method: GET
sr type: TEXT
sr url: /test/href/3
sr query: (null)
sr body: (null)
XRLT_STATUS_SUBREQUEST
sr id: 3
sr method: GET
sr type: TEXT
sr url: /test/href/2
sr query: (null)
sr body: (null)
XRLT_STATUS_SUBREQUEST
sr id: 4
sr method: GET
sr type: TEXT
sr url: /test/href/14
sr query: (null)
sr body: (null)
XRLT_STATUS_WAITING
XRLT_STATUS_SUBREQUEST
sr id: 5
sr method: GET
sr type: TEXT
sr url: /test/href/6
sr query: (null)
sr body: (null)
XRLT_STATUS_SUBREQUEST
sr id: 6
sr method: GET
sr type: TEXT
sr url: /test/href/9
sr query: (null)
sr body: (null)
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_SUBREQUEST
sr id: 7
sr method: GET
sr type: TEXT
sr url: /test/href/4
sr query: (null)
sr body: (null)
XRLT_STATUS_SUBREQUEST
sr id: 8
sr method: GET
sr type: TEXT
sr url: /test/href/5
sr query: (null)
sr body: (null)
XRLT_STATUS_WAITING
XRLT_STATUS_SUBREQUEST
sr id: 9
sr method: GET
sr type: TEXT
sr url: /test/href/7
sr query: (null)
sr body: (null)
XRLT_STATUS_WAITING
XRLT_STATUS_SUBREQUEST
sr id: 10
sr method: GET
sr type: TEXT
sr url: /test/href/8
sr query: (null)
sr body: (null)
XRLT_STATUS_WAITING
XRLT_STATUS_SUBREQUEST
sr id: 11
sr method: GET
sr type: TEXT
sr url: /test/href/13
sr query: (null)
sr body: (null)
XRLT_STATUS_WAITING
XRLT_STATUS_SUBREQUEST
sr id: 12
sr method: GET
sr type: TEXT
sr url: /test/href/15
sr query: (null)
sr body: (null)
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_SUBREQUEST
sr id: 13
sr method: POST
sr type: JSON
sr header: hehe: vavavava
sr header: hoho: veve
sr url: /some/href
sr query: papapa=pa%3Bva
sr body: super body
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_CHUNK
chunk: bb
XRLT_STATUS_DONE
